### ISSUE TITLE
Unable to pipe together build export and build import commands

### ISSUE THREAD

**Issue by edkellena** [convox/rack#1609]
Trying to tie together the build export and build import commands as suggested in the docs (https://convox.com/docs/builds/):

```sh
convox builds export $(eval "convox builds | awk 'NR==2 {print \$1}'") -a appName --rack development | convox builds import -a appName --rack production
```

(The eval line is simply grabbing the most recent build id from the `convox builds` command to pass to the export, and that bit appears to be fine.  It’s just the import)

but when running I get an EOR error on the importing side:

```
Exporting BURAYXHWSQO... OK
Uploading: 244 B / 251.83 MB [>--------------------------]   0.00 % 1s
Importing build... ERROR: EOF
```

It works fine if I split the export and import up and store the build in an intermediary file, but just not when piping them together...

**Comment by soulshake** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Build import/export flakiness](https://trello.com/c/T5v6zHd0/1843-build-import-export-flakiness)

**Comment by MiguelMoll** [comment]
This is easily reproducible on latest rack so I'll see if I can track it down. In the mean time, does saving it do disk then reading it back not work for your CI system?

**Comment by edkellena** [comment]
It does, but obviously would like to be neater/quicker/more succinct if possible :)

**Comment by MiguelMoll** [comment]
Fix for this is now published in version `20170327214533`

PR: #2088
